src/tape/ensemble_frame.py

```diff
@@ -1306,10 +1299,11 @@ def from_dask_dataframe(cl, df, ensemble=None):
 # underlying data will be resolved as EnsembleFrames or EnsembleSeries as their parrallel
 # counterparts. The underlying Dask Dataframe _meta will be a TapeSeries or TapeFrame.
 
-get_parallel_type.register(TapeSeries, lambda _: EnsembleSeries)
-get_parallel_type.register(TapeFrame, lambda _: EnsembleFrame)
-get_parallel_type.register(TapeObjectFrame, lambda _: ObjectFrame)
-get_parallel_type.register(TapeSourceFrame, lambda _: SourceFrame)
+
+get_collection_type.register(TapeSeries, lambda _: EnsembleSeries)
```

Comment:
Since `get_collection_type` isn't referenced in https://docs.dask.org/en/latest/dataframe-extend.html, it would probably be helpful to update the comment to say why we're using these instead

Reply:
Done.